Description: use system glext.h instead of the bundled SDL one
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2019-11-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gl_render.c
+++ b/gl_render.c
@@ -5,6 +5,7 @@
 #include "wavefront.h"
 #include "gl_render.h"

+#define __gl_glext_h_
 #include <SDL_opengl.h>

 static GLint lname[8]={GL_LIGHT0,GL_LIGHT1,GL_LIGHT2,GL_LIGHT3,
--- a/main.c
+++ b/main.c
@@ -3,6 +3,7 @@
  */

 #include <SDL.h>
+#define __gl_glext_h_
 #include <SDL_opengl.h>
 #include <stdlib.h>
 #include <stdio.h>
